package.json

```diff
@@ -150,7 +150,7 @@
 		"predev": "npm run check-engines",
 		"dev": "npm run build:packages && concurrently \"cross-env webpack --watch\" \"npm run dev:packages\"",
 		"dev:packages": "node ./bin/packages/watch.js",
-		"docs:generate-data-reference": "node docs/data/tool",
+		"docs:build": "node docs/tool",
```

Comment:
We have both `docs:build` and `build:packages` scripts. We should be consistent with our naming.